src/fides/api/ctl/database/database.py

```diff
@@ -60,6 +83,154 @@ def create_db_if_not_exists(database_url: str) -> None:
         create_database(database_url)
 
 
+async def load_default_policies() -> None:
+    """
+    Checks whether DSR execution policies exist in the database, and
+    inserts them to target a default set of data categories if not.
+    """
+    FIDESOPS_AUTOGENERATED_CLIENT_KEY = "fides_autogenerated_client"
+    FIDESOPS_AUTOGENERATED_STORAGE_KEY = "fidesops_autogenerated_storage_destination"
+    STRING_REWRITE_STRATEGY_NAME = "string_rewrite"
+    db_session = sync_session()
+
+    client = ClientDetail.get_by(
+        db=db_session,
+        field="fides_key",
+        value=FIDESOPS_AUTOGENERATED_CLIENT_KEY,
+    )
+    if not client:
+        client, _ = ClientDetail.create_client_and_secret(
+            db=db_session,
+            client_id_byte_length=CONFIG.security.oauth_client_id_length_bytes,
+            client_secret_byte_length=CONFIG.security.oauth_client_secret_length_bytes,
+            fides_key="fides_autogenerated_client",
+        )
+
+    client_id = client.id
+    default_data_categories = [
```

Comment:
I'd recommend a comment here like:
```
# Match all default user data categories, with the exception of...
```

In fact, since we have the default categories as an import... can we just do that? Basically the Python version of:
```
target_categories = DEFAULT_CATEGORIES - ["user.financial", "user.payment", "user.credentials"]`

Reply:
fixing